10-JavaScript/03-objects-and-inheritance.md

```diff
@@ -1,12 +1,55 @@
-# JavaScipt 103: Objects & Inheritance.
+# JavaScript 103: Objects
 ## Learning Goals
-- Compare Ruby's _classical inheritance_ with JavaScript's _prototypal inheritance_
-- Refactor some code to inherit behavior from a prototype
+- Examine how objects are created and instantiated
+- See different ways to create and extend objects
+- See how there is not a direct path from Ruby to JavaScript objects
 
+
+## Objects Practice
+We spoke briefly about objects in our introductory conversation, so let's take a deeper look.
+
+Check out this code. It shows how we would use Ruby to create a new class and create an object instance.
+```ruby
+class FuzzyPet
+  NAP_HOURS = 7
+
+  def self.dog
+    puts "borf!"
+  end
+
+  def self.cat
+    puts "mrow!"
+  end
+end
+
+FuzzyPet.NAP_HOURS  # 7
+FuzzyPet.dog        # 'borf!'
+```
+
+Now, let's try the same thing with JavaScript! hmmmm, how do we do this?
+
+```javascript
+var FuzzyPet = function(){
```

Comment:
Does this code work? I think FuzzyPet just wants to be an object, not a function.